### PR TITLE
feat(doc): add host certificates collector and analyzer

### PR DESCRIPTION
**Pull request by DexterYan**
- add host certificates validity collector and analyzer

related Issue
https://github.com/replicatedhq/troubleshoot/issues/762

### CODE REVIEW

**Comment by xavpaice** [comment]
Is this not already live in https://troubleshoot.sh/docs/host-collect-analyze/certificate/?

**Review by xavpaice — CHANGES_REQUESTED** [review]
https://troubleshoot.sh/docs/host-collect-analyze/certificate/ - the existing 'certificate' collector is too close in name, it'll be a bit confusing.  Let's figure out a new name for this collector, do a quick rename, and then this will be ready when https://github.com/replicatedhq/troubleshoot/pull/1132 merged

**Comment by xavpaice** [comment]
note: this needs an additional entry in https://github.com/replicatedhq/troubleshoot.sh/blob/main/docs/gatsby-config.js also - so it gets to show up in the index

**Comment by DexterYan** [comment]
I think we can use `SSL/TLS Certificate Validity` as the collector name since we are checking the validity period of that ssl / tls cert. And TLS Certificates are more forced on checking the key pair matching.

**Comment by xavpaice** [comment]
Waiting for https://github.com/replicatedhq/troubleshoot/pull/1132 to be merged and released, then this is good to go

**Review by jonquil2002 — DISMISSED** [review]
lgtm - committed a few minor style guide changes. But well done!

**Comment by DexterYan** [comment]
Thank you @jonquil2002. I have updated the docs to reflect the change from https://github.com/replicatedhq/troubleshoot/pull/1132

File: docs/gatsby-config.js [hunk 1]
```diff
@@ -128,6 +128,7 @@ const themeOptions = {
               'host-collect-analyze/tcpPortStatus',
               'host-collect-analyze/udpPortStatus',
               'host-collect-analyze/time',
+              'host-collect-analyze/certificatesCollection',
```
Comment (CpuID) [inline docs/gatsby-config.js:131]: move this one up the list alphabetically 👍

**Review by CpuID — CHANGES_REQUESTED** [review]
minor change suggested, but otherwise ready for approval

File: docs/gatsby-config.js (hunk 1 above)
Comment (DexterYan) [reply docs/gatsby-config.js]: thank you! Changed

**Review by CpuID — APPROVED** [review]

**Review by xavpaice — APPROVED** [review]

**Pull request merged**